cli/cmd/install.go

```diff
@@ -791,7 +789,6 @@ func (options *installOptions) buildValuesWithoutIdentity(configs *pb.All) (*l5d
 	installValues.HeartbeatSchedule = options.heartbeatSchedule()
 	installValues.RestrictDashboardPrivileges = options.restrictDashboardPrivileges
 	installValues.DisableHeartBeat = options.disableHeartbeat
-	installValues.WebImage = fmt.Sprintf("%s/web", options.dockerRegistry)
```

Comment:
This image name can be overriden by using `--addon-config` and through `values.yaml` with Helm. but when a user applies the `--registry` flag, Is the expectation that the image registry value is overriden even for add-ons? We can make sure its propagated to add-ons if thats the case. @alpeb

Reply:
Yeah I guess that is the expectation, for `--registry` to apply to all the images, including add-ons.